www/unstable/userguide/calendars.md

```diff
@@ -35,14 +42,12 @@ This is the default calendar system when constructing values without explicitly
 It is designed to correspond to the calendar described in [ISO-8601](http://en.wikipedia.org/wiki/ISO_8601),
 and is equivalent to the Gregorian calendar in all respects other than the century and year-of-century values.
 
-As of Noda Time 2.0, century and year-of-century properties have been removed as they almost certainly didn't
-behave how everyone would have expected, at which point the ISO calendar system and the Gregorian calendar system
-(with a "minimum number of days in the first week" of 4) are equivalent. The separation between them in Noda Time
-is maintained for simplicity, compatibility and consistency.
-
-The ISO calendar system always has at least 4 days in the first week of the week-year, where a week always
-runs Monday to Sunday. The first week of the week-year is therefore always the one containing the first Thursday
-in the year.
+As of Noda Time 2.0, century and year-of-century properties have
+been removed as they almost certainly didn't behave how everyone
+would have expected, at which point the ISO calendar system and the
```

Comment:
I know this is pre-existing, but perhaps simplify as "everyone expected".

(also, seems like some odd rewrapping going on here?)